nibabel/nifti1.py

```diff
@@ -379,13 +381,107 @@ def write_to(self, fileobj, byteswap):
         fileobj.write(b'\x00' * (extstart + rawsize - fileobj.tell()))
 
 
+class Nifti1DicomExtension(Nifti1Extension):
+    """NIfTI1 DICOM header extension
+
+    This class is a thin wrapper around pydicom to read a binary DICOM
+    byte string. If pydicom is not available, it silently falls back to the
+    standard NiftiExtension class.
+    """
+    def __init__(self, code, content, parent_hdr=None):
+        """
+        Parameters
+        ----------
+        code : int|str
```

Comment:
Detail point, but I think these should be `int or str` : see https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#id5